### PR TITLE
Add shortcuts to switch views

### PR DESCRIPTION
**Pull request by aledeg**

### CODE REVIEW

File: p/scripts/main.js [hunk 1]
```diff
@@ -704,6 +704,29 @@ function init_shortcuts() {
 		'disable_in_input': true
 	});
 
+	shortcut.add(shortcuts.normal_view, function () {
+		$('#nav_menu_views .view_normal').get(0).click();
```
Comment (Frenzie) [inline p/scripts/main.js:708]: Hm, is this the best way? I'm inclined to think that the `?a=normal` part of the URL is in principle more stable than the specific DOM tree.

Comment (aledeg) [reply p/scripts/main.js:708]: The thing is that when you filter the view, you might want to keep the same filter while switching views. The DOM tree hold that value. That's why I went that way.

Comment (Frenzie) [reply p/scripts/main.js:708]: Ah, of course!

**Review by Frenzie — APPROVED** [review]
Lgtm.

**Pull request merged**